lib/irb/color_printer.rb

```diff
@@ -4,6 +4,10 @@
 
 module IRB
   class ColorPrinter < ::PP
+    METHOD_IS_A = Object.instance_method(:is_a?)
+    METHOD_RESPOND_TO = Object.instance_method(:respond_to?)
+    METHOD_INSPECT = Object.instance_method(:inspect)
```

Comment:
I know we already have store the method object in other places (e.g. `inspector.rb`). But I don't want to create dependencies between files just to reuse this as it's easy to find duplications later on.